src/DetailsView/Styles/detailsview.scss

```diff
@@ -866,7 +867,7 @@ div.insights-file-issue-details-dialog-container {
                 min-height: 0;
                 .target-page-changed,
                 .static-content-details-view {
-                    margin: 24px 24px 0px 24px;
+                    margin: $fastPassRightPanelMarginTop $fastPassRightPanelMarginLeft auto $fastPassRightPanelMarginRight;
```

Comment:
Should this be 
```suggestion
                    margin: $fastPassRightPanelMarginTop $fastPassRightPanelMarginRight auto $fastPassRightPanelMarginLeft;
```
instead (top, right, bottom, left)? (also at line 895 below)

(I'm thinking Dan might be right below about preferring the split versions of these properties so the matching is explicit...)

Reply:
Updated;